ci/build-test-qemu-init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -xeuo pipefail
+
+dn=$(dirname $0)
+
+# Prow jobs don't support adding emptydir today
+export COSA_SKIP_OVERLAY=1
+
+# Create a temporary cosa workdir if COSA_DIR is not set.
+cosa_dir="${COSA_DIR:-$(mktemp -d)}"
+echo "Using $cosa_dir for build"
```

Comment:
maybe try add this to see if it works:
`git config --global --add safe.directory ${cosa_dir}`